README.md

```diff
@@ -49,7 +49,8 @@ frameworks contributed by the broader MLPerf community.
 
 * [datasets](./datasets) - Contains scripts to prepare test and calibration data used for accuracy evaluation and model quantization
 * [docs](./docs) - contains documentation
-* [flutter](./flutter) - Contains the Flutter (cross-platform) version of the app
+* [flutter](./flutter) - Contains the Flutter (iOS/Android/Windows) version of the app (for running the benchmarks on a certain device)
+* [benchmarks_web_viewer](./benchmarks_web_viewer) - Contains the React version of the app (for viewing the benchmark results on a website)
```

Comment:
@RSMNYS 
Can you rename the `benchmarks_web_viewer` directory to `react` to match with `flutter` directory naming style? Please also remember to update its reference in other places.
Maybe we need to have better names for those 2 directories, but since renaming the `flutter` directory will require plenty of changes, so we should do that in a separated PR.

Reply:
sure, let me do this